autogpt/agent/agent.py

```diff
@@ -65,24 +65,22 @@ def __init__(
         system_prompt: str,
         triggering_prompt: str,
         workspace_directory: str,
-        config: Config,
```

Comment:
I don't know why the `config` changed to `cfg`. It seems less clear to me for no real benefit. And it creates a lot of change.